src/routes/index.ts

```diff
@@ -1,7 +1,181 @@
 import express from 'express';
+import Papa from 'papaparse';
+
+import { ClientError } from 'lib/errors';
+import Notifications from '../lib/notificaitons';
+import Invite from '../modules/invite';
+import Subscribe from '../modules/subscribe';
+
+import env from '../config/env';
+import logger from '../lib/logger';
 
 const router = express.Router();
 
-router.get('/hello-world', (req, res) => res.send('Hello world!'));
+export interface InviteeData {
+    email: string;
+    fName: string;
+    lName: string;
+}
+
+export interface InviteManyData {
+    inviteeList: Array<InviteeData>;
+    MoC: string;
+    topic: string;
+    eventDateTime: string;
+    constituentScope: string;
+    region: string;
+    deliveryTime?: string; // ISO format
+}
+
+router.post('/invite', (req, res, next) => {
```

Comment:
I think the majority of this code belongs in the `module` folder and parts of this can be broken up into individual functions.  We can discuss more in our next meeting.

In general, the functions in the routes themselves are pretty much controllers.  Things that go in controllers are, in a sense, control flow logic.  For example, details like if there's no date we default to making to delivering right away is an internal detail specific to the module itself.

It might help to think of it like this.  Say we have a different type of invite in the future that is an "admin" invite.  What logic belongs here versus in the module, so we can use the same invitation logic overall.  That may help clear up what's specific to the controller versus the module.  If it doesn't that's okay.  It's easier to talk about verbally.